sources/packages/backend/libs/integrations/src/services/atbc/models/atbc.model.ts

```diff
@@ -18,17 +12,15 @@ export interface ATBCCreateClientPayload {
   email: string;
 }
 
-export interface ATBCPDCheckerPayload {
-  id: string;
-}
-
-export interface ATBCPDCheckerResponse {
-  id: string;
-  e9yStatusId: number;
-  e9yStatus: string;
+/**
+ * ATBC disability status Response.
+ */
+export interface ATBCDisabilityStatusResponse {
+  SIN: string;
+  APP_LAST_NAME: string;
+  BIRTH_DTE: string;
+  D8Y_TYPE: string;
+  D8Y_DTE: string;
 }
 
-export interface ATBCStudentModel {
-  id: number;
-  sin: string;
-}
+export const ATBC_DATE_FORMAT = "YYYY/MM/DD";
```

Comment:
Can you please explain, why we are not having the ATBC_DATE_FORMAT const in the backend/libs/utilities/src/date-utils.ts? Is there a reason, that it is created here?

Reply:
Because we never store the integration/external application specific date formats in date-utils. 

for e.g. ECE integration model

![image](https://github.com/bcgov/SIMS/assets/54600590/000dd232-6aab-4817-97fe-ab49adb78e4e)